libraries/AP_Math/scurves.h

```diff
@@ -0,0 +1,167 @@
+#pragma once
+
+#include <AP_Common/AP_Common.h>
+
+class scurves {
+
+public:
+
+    // constructors
+    scurves();
+    scurves(float tj, float Jp, float Ap, float Vp);
```

Comment:
Thanks.  We've changed this now so instead of doing the assignment we call _scurve_prev_leg.init(...).  Because there was already another method called, "init" we've renamed that one to "clear()".

Reply:
I've also added a "How to use" section at the top of SCurve.h.